### DISCUSSION TITLE
What do the values ​​shown under each station mean?

### DISCUSSION THREAD

**Discussion by azzbm** [xspanger3770/GlobalQuake#304] [Q&A]
Under each station there are some numbers, they are usually around 3, I used to think they stood for PGV (micrometers per second), but now it doesn't look like that, does anyone know what they stand for or what the units are?

**Comment by YacineBoussoufa** [comment]
From what I remember, it's a ratio of short running average and long
running average of the filtered waveform, the unit is 1.0
You can also see it in the third graph when you click on the station to
view the waveform.

--Yacine Boussoufa


Il giorno sab 8 giu 2024 alle ore 15:56 azzbm ***@***.***> ha
scritto:

**Comment by xspanger3770** [comment]
It's the ratio between the short-term average and long-term average of the signal. Neither PGV, nor PGA.
